tests/js/spec/views/settings/projectAlerts/create.spec.jsx

```diff
@@ -168,6 +168,8 @@ describe('ProjectAlertsCreate', function () {
         const {wrapper} = createWrapper({
           organization: {features: ['incidents']},
         });
+        await tick();
```

Comment:
because the project provider is getting all projects now, we have to await an extra tick and then rerender the wrapper component so that it can get past the loading state